CMakeLists.txt

```diff
@@ -22,8 +22,8 @@ FetchContent_Declare(base-n
     GIT_REPOSITORY https://github.com/azawadzki/base-n.git
     GIT_TAG 7573e77c0b9b0e8a5fb63d96dbde212c921993b4)
 FetchContent_Declare(bgfx.cmake
-    GIT_REPOSITORY https://github.com/BabylonJS/bgfx.cmake.git
-    GIT_TAG 345e3e28a9912983fc64d1dcb4da22a445afe3fd)
+    GIT_REPOSITORY https://github.com/okwasniewski/bgfx.cmake 
+    GIT_TAG 05817121430d)
```

Comment:
I know this is already on your radar, but just putting a comment so we don't forget to update this once the bgfx changes are merged.

Reply:
I will remove it once https://github.com/BabylonJS/bgfx/pull/40 gets merged. Let's leave this conversation open